src/workerd/io/compatibility-date.h

```diff
@@ -51,9 +51,33 @@ kj::Maybe<kj::String> normalizeCompatDate(kj::StringPtr date);
 // Returns the current date as a string formatted by CompatDate.
 kj::String currentDateStr();
 
-kj::Maybe<PythonSnapshotRelease::Reader> getPythonSnapshotRelease(
-    CompatibilityFlags::Reader featureFlags);
-kj::String getPythonBundleName(PythonSnapshotRelease::Reader pyodideRelease);
+class PythonRelease {
+ public:
+  PythonSnapshotRelease::Reader capnpRelease;
+  bool isDevelopment;
+  PythonRelease(PythonSnapshotRelease::Reader capnpRelease, bool isDevelopment)
+      : capnpRelease(capnpRelease),
+        isDevelopment(isDevelopment) {}
+
+  kj::StringPtr getPyodide() {
+    return capnpRelease.getPyodide();
+  }
+  kj::StringPtr getPyodideRevision() {
+    return capnpRelease.getPyodideRevision();
+  }
+  kj::StringPtr getPackages() {
+    return capnpRelease.getPackages();
+  }
+  kj::StringPtr getBaselineSnapshotHash() {
+    return capnpRelease.getBaselineSnapshotHash();
+  };
+  int64_t getBackport() {
+    return capnpRelease.getBackport();
+  }
```

Comment:
Instead of doing this, why not just have a struct with two fields: `capnpRelease` and `isDevelopment`? I think that would also solve the UaF concerns, though `PythonSnapshotRelease::Reader` is a pointer too so whatever it's pointing to needs to be kept alive.

Reply:
But I don't think we have a need for this class. See my comment above.

Reply:
I don't think that would change the UaF concerns in any way?